### PR TITLE
New branch after rebasing for evaluator module, based on PR #43

### PR DESCRIPTION
**Pull request by janez45**

### CODE REVIEW

**Comment by janez45** [comment]
Code is pretty much identical to this pull request here: https://github.com/UWARG/pathing/pull/43, rebasing gave me a lot of grief though so I created this new branch.

File: path_2024.py [hunk 1]
```diff
@@ -87,6 +88,16 @@ def main() -> int:
...
+    result, time_condition = mission_time_condition.MissionTimeCondition.create(
+        start_time, MAXIMUM_FLIGHT_TIME
+    )
+
+    if not result:
+        print("Error: Mission time condition")
+        return -1
+
+    return_to_launch_evaluator = condition_evaluator.ConditionEvaluator([time_condition])
+
```
Comment (mgupta27) [inline path_2024.py:100]: Sorry for the confusion, `path_2024.py` runs task 2 code. Since this is being tested I don't want to make any changes to it. Can we revert the changes to this file and bring back `check_stop_condition.py`. 

Also, we want to incorporate this module into `path_2024_task_1.py` instead. Can you put the mission_time_condition there.

**Review by mgupta27 — COMMENTED** [review]
Reviewed.

**Review by mgupta27 — APPROVED** [review]
LGTM
